components/data/StudentOrgsList.js

```diff
@@ -0,0 +1,100 @@
+export const Orgs = [
+	{
+		name: "Women In Computing (WINC)",
+		website: "https://winc.cs.ucr.edu/",
+		instagram: "https://www.instagram.com/wincucr/?hl=en",
+		img: "winc.png",
+		description: "club",
```

Comment:
you can remove the description off of these, since we are not using them

Reply:
oke